docs/rest/css/screen.css

```diff
@@ -1125,7 +1128,7 @@
   height: 18px;
   vertical-align: middle;
   display: inline-block;
-  background: url(/docs/rest/images/explorer_icons.png) no-repeat;
+  background: url(../images/explorer_icons.png) no-repeat;
```

Comment:
I think the absolute path is required here to ensure that this works on a production build. Can you verify that's the case?